modules/setting/setting.go

```diff
@@ -867,6 +867,16 @@ func newLogService() {
 	LogModes = strings.Split(Cfg.Section("log").Key("MODE").MustString("console"), ",")
 	LogConfigs = make([]string, len(LogModes))
 
+	useConsole := false
+	for _, mode := range LogModes {
+		if mode == "console"  {
+			useConsole = true
+		}
+	}
+	if (!useConsole) {
+		log.DelLogger("console")
```

Comment:
Yes that is correct, was thinking about setting the log level earlier but then you could miss some information about where the custom log file is read from
So that is why I have decided to do it this way